_pages/mac-mdm2.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Device Management"
+permalink: /mac-mdm2/
+author_profile: true
+sidebar:
+  nav: "nav-mac"
+header:
+  overlay_color: "#333"
+---
+
+As you didn't get the notification we can use another method to try enroll your device. 
+
+In Safari you will need to go to this website: __https://ds1506.awmdm.com/DeviceManagement/Enrollment?Gid=squa4021__
```

Comment:
Is it possible to use a URL shortener (preferably one we control) to minimize how challenging it is to type this?

Reply:
Can't we use a go link with the full https://go.sqprod.co/ ? I believe EXE was removing the restrictions so that go links are available for everyone in the world (concentrating on doing the security at the end URL not at the redirection server).